pagopa/io-app
# chore: [IOPID-1448] Fix double profile upsert

## Short description
This PR fixes the double profile upsert with the same profile version, causing a snackbar error.

This PR removes a workaround for slow devices that it was probably still there but no more necessary.

<details open><summary>Test on slow device</summary>
<p>

| Test on slow device | 
| - |
| <video src="https://github.com/pagopa/io-app/assets/16268789/142d63da-fb9b-4ea6-8000-c67ee34362eb" /> |

</p>
</details> 

## List of changes proposed in this pull request
- [ts/store/reducers/profile.ts](https://github.com/pagopa/io-app/pull/5486/files#diff-a48057473d6b62f0ea5bfaf0ed2ba3e8a390fdef1e1e3ff7e9760fd8c4a86ff6): Add `last_app_version` in the merged profile.
- [ts/sagas/startup.ts](https://github.com/pagopa/io-app/pull/5486/files#diff-8a5b2f3967d681b976fe673762bd1061f5b430130c880c1195b76af06362cf31): Remove _slow devices_ workaround.

## How to test
Run the app on slow devices and test all login flows (different profiles, first/second onboarding) and check that the mixpanel data screen is presented to the user.

Closes https://github.com/pagopa/io-app/issues/5464
